### ISSUE TITLE
All variations on by default?

### ISSUE THREAD

**Issue by csswizardry** [inuitcss/inuitcss#4]
**tl;dr** I want to get rid of the `$inuit-enable-*` variables present in every file, and just turn everything on by default.

Currently every non-standard version of an object sits behind a feature flag, like so: https://github.com/inuitcss/objects.pack/blob/f38e11d734f544ba535fded8fb2d0dd03a8718b5/_objects.pack.scss#L19-L26

My initial reason for this was so that people don’t go compiling anything and everything immediately before they need it (to stop people complaining about bloated file sizes).

For all this kinda works, there are a few issues I’m aware of:
- Lots of developers just turn everything on by default anyway, negating the point.
- Extra steps in actually getting to any code
- Not a nice experience for anyone not used to the way things work (‘Okay, let me npm install this Sass. Okay now let me compile it. Oh wait, I have to include it myself? Okay. It’s still not working, wtf?! Oh, I also have to turn it on myself as well?!’)
- Extra noise and effort at the top of each file.

I propose having everything turned on by default. inuitcss is meant to be a tiny framework, even at its largest. Adding a few more sizes of something won’t increase file size by much at all, and I feel it will improve developers’ experiences with the framework.

Further, we have tools to mitigate the effects of this: gzip eats this stuff for breakfast, and uncss et al can actually remove the CSS that doesn’t get used.
